daemon/src/projection.rs

```diff
@@ -828,26 +829,70 @@ impl Tx {
         let _ = self.quote.btc_usd.send(quote.map(|q| q.into()));
     }
 
-    fn send_order_update(&self, symbol: ContractSymbol, offer: MakerOffers) {
-        match symbol {
-            ContractSymbol::BtcUsd => {
-                self.order.btc_usd.send(offer).unwrap_or_default();
-            }
-            ContractSymbol::EthUsd => {
-                self.order.eth_usd.send(offer).unwrap_or_default();
-            }
-        }
+    fn send_offer_update(&self, offers: &Offers) -> Result<()> {
+        let OffersByContractSymbol { btc_usd, eth_usd } = split_offers_by_contract_symbol(offers);
+
+        self.order.btc_usd.send(btc_usd)?;
```

Comment:
❓ Why do we need two channels for different offers per symbol? With https://github.com/itchysats/itchysats/pull/2679 I think one channel should be enough or am I missing something?

Reply:
The PR you reference hasn't been merged yet. We'll have to adapt this once that happens.